api/apps/v1alpha1/nemo_datastore_types.go

```diff
@@ -88,20 +89,31 @@ type NemoDatastoreStatus struct {
 	State             string             `json:"state,omitempty"`
 }
 
-type NemoDataStoreParams struct {
-	AppVersion    string `json:"appVersion"`
-	GiteaEndpoint string `json:"giteaEndpoint"`
-	GiteaSecret   string `json:"giteaSecret"`
-	DatabaseURL   string `json:"databaseURL"`
-	DatabaseHost  string `json:"databaseHost"`
-	DatabasePort  string `json:"databasePort"`
-	DBSecret      string `json:"dbSecret"`
+type NemoDatastoreParams struct {
+	DBSecret         string `json:"dbSecret"`
+	GiteaAdminSecret string `json:"giteaAdminSecret"`
 
-	EnvConfigMap string `json:"envConfigmap"`
-	EnvSecret    string `json:"envSecret"`
+	ObjectStoreSecret       string `json:"objStoreSecret"`
+	DataStoreSettingsSecret string `json:"datastoreSettingsSecret"`
+	LfsJwtSecret            string `json:"lfsJwtSecret"`
 
-	InitContainerImage   string   `json:"initContainerImage,omitempty"`
-	InitContainerCommand []string `json:"initContainerCommand,omitempty"`
+	DataStoreInitSecret         string `json:"datastoreInitSecret"`
+	DataStoreConfigSecret       string `json:"datastoreConfigSecret"`
+	DataStoreInlineConfigSecret string `json:"datastoreInlineConfigSecret"`
+
+	SshEnabled bool `json:"sshEnabled"`
+
+	Persistence *Persistence `json:"persistence,omitempty"`
+}
+
+type Persistence struct {
```

Comment:
lets use the pvc spec we have defined [here](https://github.com/NVIDIA/k8s-nim-operator/blob/main/api/apps/v1alpha1/nimcache_types.go#L132) to be consistent